ui/app/src/components/Header/AccountMenu.tsx

```diff
@@ -0,0 +1,70 @@
+// Copyright 2024 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React, { MouseEvent, useState } from 'react';
+import { Box, Divider, IconButton, ListItemIcon, Menu as MUIMenu, MenuItem } from '@mui/material';
```

Comment:
I think we can keep Menu, there is no conflict with other component

```suggestion
import { Box, Divider, IconButton, ListItemIcon, Menu, MenuItem } from '@mui/material';
```